packages/lab/src/list-next/ListNext.tsx

```diff
@@ -22,6 +22,8 @@ export interface ListNextProps extends HTMLAttributes<HTMLUListElement> {
    */
   disabled?: boolean;
   /* Value for the uncontrolled version. */
+  highlightedIndex?: number;
```

Comment:
Should the comment be 'Value for the controlled version'?

Reply:
oh true, it is the other way around :P will change it